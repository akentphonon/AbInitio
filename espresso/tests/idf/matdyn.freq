 &plot nbnd=   9, nks=  84 /
            0.000000  0.000000  0.000000
   -0.0000   -0.0000    0.0000  337.6640  337.6640  397.4299
  563.6032  563.6032  686.9338
            0.000000  0.000000  0.088045
   51.1814   51.1814   91.7587  333.3738  333.3738  398.0956
  561.9823  561.9823  684.7549
            0.000000  0.000000  0.176091
   97.4372   97.4372  173.0758  318.6332  318.6332  400.6004
  553.6947  553.6947  672.5496
            0.000000  0.000000  0.264136
  142.1108  142.1108  238.3742  293.6918  293.6918  401.2963
  536.3476  536.3476  649.8923
            0.000000  0.000000  0.352182
  187.8003  187.8003  263.5324  263.5324  288.3643  392.0013
  517.4742  517.4742  631.2234
            0.000000  0.000000 -0.440227
  228.6295  228.6295  229.8004  229.8004  312.8752  380.8514
  509.2911  509.2911  625.2289
            0.000000  0.115470  0.000000
   68.1868  100.3926  149.6003  346.8262  361.0442  392.8633
  543.0440  565.9867  682.1019
            0.000000  0.115470  0.088045
  106.9353  111.9419  161.2030  342.7604  355.6962  395.7250
  539.9328  563.8974  679.6917
            0.000000  0.115470  0.176091
  138.6925  153.2747  201.6410  329.5646  342.0771  401.1074
  531.3961  552.4052  668.9415
            0.000000  0.115470  0.264136
  172.2347  185.1481  252.4816  307.7805  323.9968  402.9124
  523.0864  532.9937  650.4165
            0.000000  0.115470  0.352182
  208.0923  215.8157  280.5184  281.5890  316.2265  395.0940
  520.5758  522.6414  633.9165
            0.000000  0.115470 -0.440227
  232.1358  233.6348  261.7708  274.9221  328.1975  385.8895
  518.2097  525.2833  627.4189
            0.000000  0.230940  0.000000
  132.4007  178.4940  246.9832  375.7272  381.0700  445.5426
  621.4798  638.2632  667.8450
            0.000000  0.230940  0.088045
  153.0223  183.7285  251.6357  372.9571  381.8613  446.2053
  617.4875  637.4518  667.4480
            0.000000  0.230940  0.176091
  189.7728  197.7178  264.7011  364.7170  385.1528  447.3621
  611.5690  629.2390  664.8577
            0.000000  0.230940  0.264136
  216.3719  218.4650  279.6406  352.4025  391.6434  447.0772
  611.7045  616.3345  660.7740
            0.000000  0.230940  0.352182
  233.5193  238.3910  287.5144  340.1909  398.9239  444.2707
  613.5925  619.5514  655.6299
            0.000000  0.230940 -0.440227
  240.9047  246.8052  288.8732  334.6989  402.3074  442.1227
  616.3036  626.6758  651.0611
            0.000000  0.346410  0.000000
  199.8276  224.5766  295.8801  372.3699  417.8817  564.2559
  637.0625  720.2942  756.2020
            0.000000  0.346410  0.088045
  208.9808  227.0789  296.6920  377.9278  416.8247  565.4450
  632.3050  721.4146  759.5879
            0.000000  0.346410  0.176091
  227.1980  233.5891  296.7359  392.7950  413.5454  567.6251
  621.8835  719.2970  768.0954
            0.000000  0.346410  0.264136
  241.7553  244.0177  293.4085  408.9812  410.2045  569.9590
  613.0187  715.2479  777.9679
            0.000000  0.346410  0.352182
  248.2486  258.5357  287.3944  405.6404  422.2683  572.8853
  609.6447  719.5533  785.1113
            0.000000  0.346410 -0.440227
  250.5856  267.0376  282.3221  404.6338  426.2385  574.4929
  609.4076  724.6291  787.5354
            0.000000  0.461880  0.000000
  247.9964  248.8622  320.6327  383.2732  451.8014  564.5909
  668.2227  729.5684  786.0113
            0.000000  0.461880  0.088045
  249.0833  251.6459  317.8811  392.3832  451.7601  562.9830
  671.0562  732.1807  788.6512
            0.000000  0.461880  0.176091
  251.5950  257.8088  310.6105  413.3993  451.5142  560.0908
  675.8409  732.0712  793.4213
            0.000000  0.461880  0.264136
  254.3290  264.5309  301.6630  434.7667  451.1006  559.7915
  679.0056  729.4096  797.7935
            0.000000  0.461880  0.352182
  256.2481  271.1319  294.5816  448.8662  450.8574  563.0408
  681.4557  736.6542  802.8882
            0.000000  0.461880 -0.440227
  256.8650  274.7591  291.5080  450.8378  453.5701  565.2774
  682.7134  743.7576  805.7397
            0.000000 -0.577350  0.000000
  255.1737  262.6450  327.8526  423.0210  459.1549  487.0553
  723.2986  725.6772  756.6650
            0.000000 -0.577350  0.088045
  255.6922  263.7016  325.0587  430.6779  459.3818  489.0921
  726.9932  730.5607  757.3597
            0.000000 -0.577350  0.176091
  256.7933  266.1277  317.8517  448.1043  459.9633  495.9781
  728.3119  739.4555  757.7637
            0.000000 -0.577350  0.264136
  257.9009  268.6377  310.0277  460.6759  466.0879  506.5253
  725.2739  746.3300  757.2619
            0.000000 -0.577350  0.352182
  258.5730  270.6910  305.6728  461.2662  479.1194  515.7213
  731.1155  751.9498  757.8555
            0.000000 -0.577350 -0.440227
  258.7285  271.5995  304.6865  461.4985  483.9768  519.2174
  737.5704  754.7210  758.6600
            0.100000  0.173205  0.000000
  114.2604  165.5753  226.3658  371.5654  384.1758  413.7413
  597.8868  609.1031  671.9066
            0.100000  0.173205  0.088045
  138.9188  171.7531  232.2139  368.2734  382.2959  416.0305
  595.2563  607.1564  670.2723
            0.100000  0.173205  0.176091
  180.6855  187.7908  250.1461  358.8201  379.4356  419.9334
  590.4062  597.6846  664.0123
            0.100000  0.173205  0.264136
  208.9269  211.8770  272.9719  344.7157  379.4997  420.4218
  582.8640  591.8048  653.6151
            0.100000  0.173205  0.352182
  229.4042  233.2280  286.2204  329.6646  385.9572  413.3938
  574.2357  605.2957  641.3620
            0.100000  0.173205 -0.440227
  238.9924  242.0912  288.9964  322.1681  396.5281  402.3473
  572.8825  622.5519  627.4697
            0.100000  0.288675  0.000000
  175.8864  215.9420  280.8851  373.7086  422.5222  512.8745
  649.9077  707.0418  741.4638
            0.100000  0.288675  0.088045
  188.5072  218.9434  282.8655  377.6205  421.1054  513.6866
  644.7548  706.3474  746.0701
            0.100000  0.288675  0.176091
  213.3634  226.9881  287.1663  388.7170  417.2047  515.5365
  634.0180  703.3995  756.0818
            0.100000  0.288675  0.264136
  235.0923  237.4394  289.5096  402.6534  411.6137  517.6665
  624.4252  697.5805  766.7338
            0.100000  0.288675  0.352182
  246.2974  250.9902  287.6376  405.4473  412.9872  519.6313
  618.6300  691.6134  775.1033
            0.100000  0.288675 -0.440227
  249.8590  257.9967  285.2439  402.8658  416.1077  520.5223
  616.7645  689.0692  778.4372
            0.100000  0.404145  0.000000
  233.5590  244.3859  310.3151  375.3447  467.3963  601.6073
  619.6589  749.4233  802.9089
            0.100000  0.404145  0.088045
  238.0882  245.8696  308.7332  383.3086  467.0956  598.4070
  621.4064  749.6626  806.6786
            0.100000  0.404145  0.176091
  247.9872  249.7252  303.8810  402.2427  465.9967  592.1238
  623.9961  747.0434  814.2821
            0.100000  0.404145  0.264136
  254.3749  258.7505  296.4947  421.9868  464.2551  587.3178
  626.1241  742.5108  820.6990
            0.100000  0.404145  0.352182
  258.1409  269.1856  288.2913  435.2296  462.8745  585.4375
  629.0190  742.8606  825.0893
            0.100000  0.404145 -0.440227
  259.4811  277.8226  280.8030  439.6912  462.4265  585.1694
  630.7216  744.8726  826.9150
            0.100000  0.519615  0.000000
  257.9316  260.8196  322.1594  405.8888  484.4249  517.1853
  700.2661  738.4790  767.5434
            0.100000  0.519615  0.088045
  258.6516  262.0802  319.6890  414.0237  484.5628  517.8782
  703.1154  739.7216  769.1189
            0.100000  0.519615  0.176091
  260.3969  264.9610  313.4697  432.7000  484.6766  520.4755
  707.7166  736.4621  771.7304
            0.100000  0.519615  0.264136
  262.3134  268.1803  306.7002  451.9309  484.7108  525.4462
  710.4271  729.9134  773.9780
            0.100000  0.519615  0.352182
  263.8357  270.9387  302.6879  465.3643  485.0163  531.1348
  712.3892  733.2880  777.3674
            0.100000  0.519615 -0.440227
  264.4486  272.1112  301.6486  470.1715  485.2437  533.7585
  713.6582  738.3980  779.5472
            0.200000  0.346410  0.000000
  226.3418  245.5594  304.2336  373.5984  500.9709  585.5185
  612.6207  767.0257  822.3402
            0.200000  0.346410  0.088045
  231.6650  247.0319  303.1558  380.8755  500.6137  586.1617
  609.6980  765.7133  827.1742
            0.200000  0.346410  0.176091
  243.5713  251.2061  299.8752  398.3606  499.2842  587.6226
  603.7648  762.6214  836.8665
            0.200000  0.346410  0.264136
  256.2541  256.9692  294.1057  416.8099  497.0790  589.9024
  597.9603  757.1377  844.1298
            0.200000  0.346410  0.352182
  262.0407  267.3448  286.3586  429.2430  495.0779  592.7162
  592.9668  748.9721  847.1948
            0.200000  0.346410 -0.440227
  264.0497  274.9662  279.5983  433.4295  494.3133  590.2491
  594.5287  744.5249  847.8730
            0.200000  0.461880  0.000000
  257.2107  264.8258  312.3639  389.4338  545.6388  552.7983
  659.3351  765.7956  788.1515
            0.200000  0.461880  0.088045
  258.9142  265.7702  310.4167  397.9150  545.3210  553.0815
  660.1753  764.2343  790.3895
            0.200000  0.461880  0.176091
  263.1105  268.2515  305.8620  417.1678  545.1167  553.2931
  661.7613  757.2692  794.3815
            0.200000  0.461880  0.264136
  268.1234  271.6067  300.5889  436.7639  545.8333  553.0740
  663.3782  747.2286  796.8092
            0.200000  0.461880  0.352182
  272.3915  274.6287  296.5001  450.3063  547.0704  552.9773
  665.0931  742.2896  798.1831
            0.200000  0.461880 -0.440227
  274.1697  275.8071  295.0216  455.0937  547.6883  553.0417
  665.9486  742.0080  798.8484
            0.200000  0.577350  0.000000
  265.5350  272.1565  312.9189  417.8878  483.9900  562.5407
  717.4264  727.1459  757.9763
            0.200000  0.577350  0.088045
  266.1726  272.8977  311.8286  425.3788  486.5548  562.9543
  719.2277  727.0656  756.8277
            0.200000  0.577350  0.176091
  267.6751  274.7159  309.0062  443.0232  492.9745  563.6102
  721.9286  726.3726  750.3813
            0.200000  0.577350  0.264136
  269.3334  277.1071  306.1938  461.6677  500.8132  563.9362
  721.0792  725.9776  742.9452
            0.200000  0.577350  0.352182
  270.9839  279.2907  305.0342  474.8606  507.5683  564.1956
  719.0862  727.4490  742.6972
            0.200000  0.577350 -0.440227
  271.8762  280.1213  304.9459  479.5721  510.3426  564.3703
  719.4314  728.7016  743.9903
            0.300000  0.519615  0.000000
  266.4658  283.6612  303.3406  411.5778  473.7426  629.8450
  685.8129  732.0055  767.8564
            0.300000  0.519615  0.088045
  267.0474  284.3641  303.0004  419.2960  477.6243  630.2830
  686.0699  732.4048  766.4849
            0.300000  0.519615  0.176091
  268.4218  286.0856  302.1307  437.6409  487.1133  631.2543
  686.5469  732.9603  760.9001
            0.300000  0.519615  0.264136
  270.1914  288.3920  301.3644  457.0251  497.7117  632.2219
  686.7893  733.3844  751.3854
            0.300000  0.519615  0.352182
  272.0456  290.7646  301.2978  470.3852  505.5722  632.9372
  686.9755  734.2429  742.7572
            0.300000  0.519615 -0.440227
  272.9227  291.8521  301.4620  474.9956  508.4414  633.2183
  687.1597  734.8325  739.4130
