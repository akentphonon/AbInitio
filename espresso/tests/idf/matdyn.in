&INPUT
    asr = 'simple',
    amass(1) = 24.305,
    amass(2) = 11.000,
    flfrc = 'mgb2666.fc',
    nk1 = 10,
    nk2 = 10,
    nk3 = 10,
    dos = .true.,
/
