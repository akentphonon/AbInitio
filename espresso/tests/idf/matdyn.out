     Message from routine matdyn:
     Z* not found in file mgb2666.fc, TO-LO splitting at q=0 will be absent!
     A direction for q was not specified:TO-LO splitting will be absent
